core/vm/evm.go

```diff
@@ -337,6 +345,14 @@ func (evm *EVM) DelegateCall(caller ContractRef, addr common.Address, input []by
 	}
 	snapshot := evm.IntraBlockState.Snapshot()
 
+	// Capture the tracer start/end events in debug mode
+	if evm.vmConfig.Debug {
+		_ = evm.vmConfig.Tracer.CaptureStart(evm.depth, caller.Address(), addr, false, input, gas, big.NewInt(-1))
```

Comment:
Forgot to mention: in here there is no `value` around, so I set it to -1 just to let callees notice the specific case. Let me know whether this should be dealt with in another way.